client/less/challenge.less

```diff
@@ -1,231 +1,243 @@
 .challenges-instructions-panel {
-  clear: both;
-  overflow-x: hidden;
-  overflow-y: auto;
-  padding-left: 5px;
-  padding-right: 5px;
-  margin-bottom: 10px;
+    clear: both;
```

Comment:
Remove reformatting changes. PR's should be scoped the issue it tries to solve. This adds a ton of noise and makes the PR hard to review.